flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -429,6 +444,30 @@ public Builder<T> setAll(Map<String, String> properties) {
       return this;
     }
 
+    /**
+     * Emits watermarks once per split based on the file statistics for the given split. The
```

Comment:
nit: `based on the file statistics` -> `based on the min value of column statistics from file metadata`